src/views/show-control/SiteSurveyButton.tsx

```diff
@@ -0,0 +1,99 @@
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+
+// @ts-ignore
```

Comment:
I don't think we need `@ts-ignore` here. Typings are provded in `types/` and the TS compiler and the linter should pick them up (it is doing so for me in VS Code).